### PR TITLE
[luci] Add ResolveCustomOpBatchMatMulPass test

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit adds ResolveCustomOpBatchMatMulPass test.

ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/luci/pass/src/ResolveCustomOpBatchMatMulPass.test.cpp [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  EXPECT_NE(nullptr, input_0);
+  EXPECT_NE(nullptr, input_1);
+}
+
+TEST_F(BatchMatmulV2Graph, wrong_condition_NEG)
+{
+  luci::ResolveCustomOpBatchMatMulPass pass;
+
+  // wrong custom code
+  _batchmatmulv2->custom_code("BatchMatMulv2");
```
Comment (seanshpark) [inline compiler/luci/pass/src/ResolveCustomOpBatchMatMulPass.test.cpp:142]: `v` is lower case :)

Comment (mhs4670go) [reply compiler/luci/pass/src/ResolveCustomOpBatchMatMulPass.test.cpp:142]: Right. It is negative case.

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**